src/sentry/replays/query.py

```diff
@@ -596,6 +652,33 @@ def _activity_score():
 }
 
 
+TAG_QUERY_ALIAS_COLUMN_MAP = {
+    "replay_type": Column("replay_type"),
+    "platform": Column("platform"),
+    "environment": Column("environment"),
+    "release": Column("release"),
+    "dist": Column("dist"),
+    "url": Function("arrayJoin", parameters=[Column("urls")]),
```

Comment:
Could be helpful to document how arrayJoin changes the behavior of the query (flattens the array and expands the returned rowset) or just a link to the documentation.